Makefile

```diff
@@ -31,10 +31,11 @@ install-provisioner:
 init-terraform:
 	terraform init -upgrade=true
 
-ssh-config: export SSH_CONFIG_DIR := ~/.ssh/config.d
-ssh-config: export SSH_CONFIG_FILE := infra-nimbus
-ssh-config: export SSH_USERNAME := $$(whoami)
-	scripts/create-ssh-config.sh
+ssh-config:
+	SSH_CONFIG_DIR=~/.ssh/config.d \
+	SSH_CONFIG_FILE=infra-nimbus \
+	SSH_USERNAME="$$(whoami)" \
```

Comment:
Oh, I see, when this change was made in #32 it was missing the last `ssh-config:`. Guess Zah didn't check if it works.

But I don't see the reason to change it like that. The rest looks fine but this can be fixed easier:
```makefile
ssh-config: export SSH_CONFIG_DIR ?= ~/.ssh/config.d
ssh-config: export SSH_CONFIG_FILE ?= infra-nimbus
ssh-config: export SSH_USERNAME ?= $$(whoami)
ssh-config:
	scripts/create-ssh-config.sh
```
This way it is fixed and also allows to override values.

Reply:
Mind if I amend the commit?

Reply:
> I don't see the reason to change it like that

Shell variables are easier to deal with than Makefile variables and that weird abuse of target dependencies.

Using custom values can be done with the shell script itself.

Reply:
> Mind if I amend the commit?

Go ahead.

Reply:
That's true, the script allows it, but there's no harm in making it possible to be overriden via `make`.

And it's not abuse, it's normal functionality: https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html